moveit_core/robot_trajectory/CMakeLists.txt

```diff
@@ -1,13 +1,13 @@
 set(MOVEIT_LIB_NAME moveit_robot_trajectory)
 
 add_library(${MOVEIT_LIB_NAME} src/robot_trajectory.cpp)
-set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
+set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION ${${PROJECT_NAME}_VERSION})
 
-target_link_libraries(${MOVEIT_LIB_NAME} moveit_robot_model moveit_robot_state ${catkin_LIBRARIES} ${urdfdom_LIBRARIES} ${urdfdom_headers_LIBRARIES} ${Boost_LIBRARIES})
-add_dependencies(${MOVEIT_LIB_NAME} ${catkin_EXPORTED_TARGETS})
+target_link_libraries(${MOVEIT_LIB_NAME} moveit_robot_model moveit_robot_state ${rclcpp_LIBRARIES} ${rmw_implementation_LIBRARIES} ${urdfdom_LIBRARIES} ${urdfdom_headers_LIBRARIES} ${Boost_LIBRARIES})
+# add_dependencies(${MOVEIT_LIB_NAME} ${catkin_EXPORTED_TARGETS})
 
 install(TARGETS ${MOVEIT_LIB_NAME}
-  ARCHIVE  DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
-  LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION})
+  ARCHIVE  DESTINATION lib
```

Comment:
I wish I had made a point of this earlier discussing the install locations for rqt2 with @dirk-thomas he strongly urged us to install files into namespaced sub-folders. If my memory serves me correctly, this is not that important unless you are installing with the `--merge-install` flag which Windows users are instructed to do. Namespaced sub-folders ensure that there are no collisions between libraries. I believe that using subfolders may have implications for plugins but agian, @dirk-thomas may correct me on that point. Perhaps this is something we will want to add to the roadmap for the future or maybe it is important now. Seeing as the majority of ROS2 packages do not use the sub-directory approach, I am inclined to say that we can put this on the roadmap for later.

Reply:
> this is not that important unless you are installing with the `--merge-install` flag

The same is the case when you release the package. If two packages would install the same header (because they both didn't namespace it) both Debian packages can't be installed at the same time.

> Seeing as the majority of ROS2 packages do not use the sub-directory approach, I am inclined to say that we can put this on the roadmap for later.

I am not sure which ROS 2 packages you are referring to but I do think that all ROS 2 packages only install headers in a subdirectory named after package.